server/checks/PullRequest.js

```diff
@@ -0,0 +1,99 @@
+import Check from './Check'
+import { logger } from '../../common/debug'
+import * as EVENTS from '../model/GithubEvents'
+
+const CHECK_TYPE = 'pullrequest'
+const ACTIONS = ['opened', 'edited', 'reopened', 'synchronize']
+
+const ISSUE_PATTERN = /^[^#\s]*#\d+$/
+// Grubber's pattern
+const URL_PATTERN = /\b((?:[a-z][\w-]+:(?:\/{1,3}|[a-z0-9%])|www\d{0,3}[.]|[a-z0-9.\-]+[.][a-z]{2,4}\/)(?:[^\s()<>]+|\(([^\s()<>]+|(\([^\s()<>]+\)))*\))+(?:\(([^\s()<>]+|(\([^\s()<>]+\)))*\)|[^\s`!()\[\]{};:'".,<>?«»“”‘’]))/i
+
+const info = logger(CHECK_TYPE, 'info')
+const error = logger(CHECK_TYPE, 'error')
+
+const status = (description, state = 'success') => ({
+  description,
+  state,
+  context: 'zappr/check/pull-request'
+})
+
+const isLongEnough = (str, requiredLength) => (str || '').length > requiredLength
+const containsPattern = pattern => str => (str || '').split(' ')
```

Comment:
You could use `some`, ie `str.split(' ').some(s => pattern.test(s))`. Any special reason why you split the string on whitespace instead of just matching the whole string?